airflow/providers/amazon/aws/hooks/eventbridge.py

```diff
@@ -16,12 +16,75 @@
 # under the License.
 from __future__ import annotations
 
+import json
+
 from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
+from airflow.providers.amazon.aws.utils import trim_none_values
+
+
+def validate_json(pattern: str) -> None:
```

Comment:
lets mark private?
We might move this one to utils in the future so till we decide how helper functions are organized I think it's best to set them as private to avoid deprecation cycle in the future.

```suggestion
def _validate_json(pattern: str) -> None:
```